src/components/editor/styleMap.js

```diff
@@ -1,18 +1,17 @@
 const styleMap = {
-    'MYSTYLE1': {
+    'bold': {
         fontWeight: 'bold',
-        fontStyle: 'italic',
-        regEx: /\*(.*?)\*/g,
-        keyCharsCount: 1 //This is the number of characters in the key, 1 for * and 2 for **
+        regEx: /\*{2}(?<text>.*?)\*{2}/g,
+        keyCharsCount: 2 //This is the number of characters in the key, 1 for * and 2 for **
     },
     'MYSTYLE2': {
```

Comment:
Don't need to keep MYSTYLEs in there. Go ahead and replace those with real styles when you get to it.